test/lib/global_uid_test.rb

```diff
@@ -469,19 +471,39 @@ def with_timed_out_connection(server:, end_time:)
 
     describe "with per-process_affinity" do
       before do
-        GlobalUid::Base.global_uid_options[:per_process_affinity] = true
+        GlobalUid.configuration.per_process_affinity = true
       end
 
-      it "increment sequentially" do
+      it "increment sequentially, selecting a server at random" do
         last_id = 0
         10.times do
           this_id = WithGlobalUID.create!.id
           assert_operator this_id, :>, last_id
         end
+
+        refute_empty(GlobalUid::Base.servers[0].allocator.recent_allocations)
+        assert_empty(GlobalUid::Base.servers[1].allocator.recent_allocations)
       end
 
       after do
-        GlobalUid::Base.global_uid_options[:per_process_affinity] = false
+        GlobalUid.configuration.per_process_affinity = false
+      end
+    end
+
+    describe "without per-process_affinity" do
+      before do
+        GlobalUid.configuration.per_process_affinity = false
+      end
+
+      it "increment sequentially, using the first configured id_server" do
```

Comment:
"per_process_affinity = false" is the “random” one, I think. 

The comments in configuration.rb and test_helper.rb are disagreeing on this issue.

Reply:
The spec description is wrong, great catch. I'm going to rename that option, I've been caught out a couple of times by it. It has nothing to do with `per process`, we'll either shuffle connections, or we won't and the default behaviour is to not shuffle them.